### PR TITLE
PSP-591 re-adding commented out multihop functional tests

### PR DESCRIPTION
**Pull request by mozrokafor**
## Description

    re-adding commented out multihop tests

### CODE REVIEW

File: tests/functional/testMultihop.js [hunk 1]
```diff
@@ -53,590 +39,595 @@ describe('Server list', function() {
...
+                    countryId, city.name);
+                await vpn.waitForQuery(cityId.visible());
+
+                await vpn.setQueryProperty(
+                    queries.screenHome.serverListView.COUNTRY_VIEW, 'contentY',
+                    parseInt(await vpn.getQueryProperty(cityId, 'y')) +
+                        parseInt(await vpn.getQueryProperty(countryId, 'y')));
+                await vpn.waitForQuery(cityId.visible());
+                
+                await vpn.getQueryProperty(cityId, 'radioButtonLabelText');
```
Comment (bakulf) [inline tests/functional/testMultihop.js:172]: Why this line?

File: tests/functional/testMultihop.js [hunk 2]
```diff
@@ -53,590 +39,595 @@ describe('Server list', function() {
...
+
+                await vpn.waitForQueryAndClick(cityId.visible());
+                await vpn.waitForQuery(queries.screenHome.STACKVIEW.ready());
+                await vpn.wait(1000)
```
Comment (bakulf) [inline tests/functional/testMultihop.js:176]: This should not needed. Maybe you want to check the state of some stackview or loader obj.

File: tests/functional/testMultihop.js [hunk 3]
```diff
@@ -53,590 +39,595 @@ describe('Server list', function() {
...
+                    countryId, city.name);
+                await vpn.waitForQuery(cityId.visible());
+
+                await vpn.setQueryProperty(
+                    queries.screenHome.serverListView.COUNTRY_VIEW, 'contentY',
+                    parseInt(await vpn.getQueryProperty(cityId, 'y')) +
+                        parseInt(await vpn.getQueryProperty(countryId, 'y')));
+                await vpn.waitForQuery(cityId.visible());
+                
+                await vpn.getQueryProperty(cityId, 'radioButtonLabelText');
```
Comment (bakulf) [inline tests/functional/testMultihop.js:222]: Why this?

File: tests/functional/testMultihop.js [hunk 4]
```diff
@@ -53,590 +39,595 @@ describe('Server list', function() {
...
+
+                await vpn.waitForQueryAndClick(cityId.visible());
+                await vpn.waitForQuery(queries.screenHome.STACKVIEW.ready());
+                await vpn.wait(1000)
```
Comment (bakulf) [inline tests/functional/testMultihop.js:226]: Same comment here.

File: tests/functional/testMultihop.js [hunk 5]
```diff
@@ -53,590 +39,595 @@ describe('Server list', function() {
...
+            queries.screenHome.serverListView.COUNTRY_VIEW, 'contentY',
+            parseInt(await vpn.getQueryProperty(cityOneId, 'y')) +
+                parseInt(await vpn.getQueryProperty(entryCountryId, 'y')));
+        await vpn.wait();
+
+        await vpn.waitForQueryAndClick(cityOneId.visible());
+        await vpn.waitForQuery(queries.screenHome.STACKVIEW.ready());
+
+        // Back at the main view. select the exit entries
+        await vpn.wait()
```
Comment (bakulf) [inline tests/functional/testMultihop.js:290]: This wait() can be all removed.

**Review by bakulf — CHANGES_REQUESTED** [review]
Apply clang-format please. Can we remove the `wait()`?

File: tests/functional/testMultihop.js [hunk 6]
```diff
@@ -53,590 +39,595 @@ describe('Server list', function() {
...
-       newCurrentCountry = firstServer.localizedName;
-       newCurrentCity = cityThree.localizedName;
-
-       // wait and assert server switching for multihop
-       await vpn.waitForCondition(
-           async () => {return vpn.lastNotification().title ==
+    it('opening the entry and exit server list', async () => {
+        await vpn.waitForQueryAndClick(queries.screenHome.serverListView.MULTIHOP_SELECTOR_TAB);
+        assert.equal(
+            await vpn.getQueryProperty(
```
[the remark this answers is not in the document]
Comment (mozrokafor) [reply tests/functional/testMultihop.js:46]: added, saw a few others that needed it too. will update those as well.

**Pull request closed without merging**